src/widgets.js

```diff
@@ -350,8 +350,13 @@ define([
             super_updateValue();
         };
 
+        widget._setURLHash = _.debounce(function(){
```

Comment:
The mug should be passed in here, right?
```suggestion
        widget._setURLHash = _.debounce(function (mug) {
```

Reply:
Yep. Yep. Thanks for pointing out, it was an oversight. I am wondering how it is was working then 🤔

Reply:
We should either make this change or not pass `mug` below in `widget._setURLHash(mug)`

Reply:
Agreed. We should pass `mug` as you have suggested.